Extensions/ArtifactEngine/EngineTests/artifactEngineTests.ts

```diff
@@ -51,26 +52,18 @@ describe('artifactEngine.processItems', () => {
     it('should call getArtifactItem only for artifact that match the download pattern', async () => {
         var testProvider = new providers.StubProvider();
         var downloadOptions = new engine.ArtifactEngineOptions();
-        downloadOptions.itemPattern = '*(PAth4|path5)\\**';
+        downloadOptions.itemPattern = '@(PAth4|path5)\\**';
```

Comment:
is this to validate the scenario of service passing multiple items to download for partial artifacts (the discussion we had with Lova)?

Reply:
No, it is just to test single patterns * will match Path4Path4 also.